web_src/js/features/misc.js

```diff
@@ -0,0 +1,9 @@
+import {initComboMarkdownEditor} from './comp/ComboMarkdownEditor.js';
+
+export function initComboMarkdownEditorGlobal() {
```

Comment:
What I'm wondering about:
Don't we already have such a function?
Why not enhance that instead?

Reply:
Or what exactly did we do previously to initialize them?

Reply:
Can't we remove these other functions then, and put everything into the same method?

Reply:
Agree, there should only be one init function. Should be possible to make it work there?

Reply:
I also agree that it should avoid introducing a new mechanism, it reduces the maintainability.

Reply:
the Markdown editor on the release page is now using the new class. The other locations where `initComboMarkdownEditor` is used are parts of a bigger code, so I left it as it is.